content/about/limits.md

```diff
@@ -1,20 +1,23 @@
-﻿---
+---
 title: Limits
 weight: 4
 ---
-
-## Plans
-
-Workers plans are separate from any Cloudflare plan (Free, Professional, Business) you may have. All users can access the Workers free plan subject to its [request limits](#request-limits). Upgrading to the Unlimited (Paid) plan lifts the request limits and increases CPU time limits.
-
-| Plan             | CPU Limit | Daily Request Limit | Burst Rate Limit         |
-| ---------------- | --------- | ------------------- | ------------------------ |
-| Free             | 10ms      | 100,000             | 1000 requests per minute |
-| Unlimited (Paid) | 50ms      | none                | none                     |
```

Comment:
I'd consider keeping this table on this page in a prominent position. This is an easy-to-scan summary of the most important limits.